src/gluonts/model/rotbaum/_predictor.py

```diff
@@ -168,6 +172,11 @@ def __init__(
         self.quantiles = quantiles
         self.model_list = None
 
+        logging.info(
```

Comment:
We started to create new loggers for each module:

```python
logger = logging.getLogger(__main__)
```

You can search the code to see how it is generally used.

The advantage is that this prints some more context, i.e. the module the message came from.

Reply:
+1